cmd/elasticindexer/config/config.toml

```diff
@@ -4,12 +4,6 @@
         "receipts", "scresults", "accountsesdt", "accountsesdthistory", "epochinfo", "scdeploys", "tokens", "tags",
         "logs", "delegators", "operations", "collections"
     ]
-    [config.elastic-cluster]
-        use-kibana = false
-        url = "http://localhost:9200"
-        username = ""
-        password = ""
-        bulk-request-max-size-in-bytes = 4194304 # 4MB
```

Comment:
enabled-indices and web-socket are still "prefs".. might move them as well

Reply:
moved